tuneme/templates/polls/results.html

```diff
@@ -11,20 +11,33 @@
 </div>
 
 <div class="block orange">
+	{% if question.show_results%}
 	<h4>{{ question.title }}</h4>
 	<p class="grey">{% trans "Your answers" %}</p>
-	{% for choice in choices %}
-		<div class="poll-result">
-				<div class="result-line {% if forloop.first %}winner{% endif %}">
-				<span class="option">{{ choice.title }}</span>
-				<br>
-				<span class="bar">
-	        <span style="width:{{ choice.percentage }}%"></span>
-					{{ choice.percentage }}%
-	      </span>
+		{% for choice in choices %}
+			<div class="poll-result">
+					<div class="result-line {% if forloop.first %}winner{% endif %}">
+					<span class="option">{{ choice.title }}</span>
+					<br>
+					<span class="bar">
+						{% if question.result_as_percentage %}
+			        <span style="width:{{ choice.percentage }}%"></span>
+							{{ choice.percentage }}%
+						{% else %}
+							<span style="width:{{ choice.percentage }}%"></span>
+							{% if choice.votes = 1%}
+								{{ choice.votes }} vote
+							{% else %}
+								{{ choice.votes }} votes
+							{% endif %}
```

Comment:
there's a template tag for this:
https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#pluralize